 * Serving Flask app 'main' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://172.31.34.126:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 11:34:14] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 11:38:22] "POST /register HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:38:22] "GET /profileImg/2021525323333.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 11:42:24] "POST /register HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:42:24] "GET /profileImg/2021525323333.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 11:49:48] "POST /register HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:49:49] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:53:55] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:53:55] "GET /static/img/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:54:16] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:54:16] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
175.196.169.199 - - [25/May/2021 11:54:18] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:54:18] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
175.196.169.199 - - [25/May/2021 11:54:33] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:54:33] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
175.196.169.199 - - [25/May/2021 11:54:34] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:54:34] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
175.196.169.199 - - [25/May/2021 11:54:34] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:54:34] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
175.196.169.199 - - [25/May/2021 11:55:21] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:55:21] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
175.196.169.199 - - [25/May/2021 11:57:06] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 11:59:53] "POST /register HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 11:59:53] "GET /static/img/2021525323333 HTTP/1.1" 404 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:01:21] "POST /register HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 12:01:22] "GET /static/img/2021525323333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 12:02:01] "GET /profileImg/2021525493333.jpg HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 12:02:01] "GET /static/img/2021525493333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:06:04] "POST /register HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 12:06:05] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:10:38] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:10:57] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:11:56] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:12:44] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:13:06] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:14:27] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:15:49] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:16:16] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:18:42] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:19:43] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:20:08] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:20:28] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:20:45] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:29:24] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021511344
['2021525323333', '등록된강아지', '0.9153762437973622']
등록된강아지
lookupdb에 예외가 발생했습니다
20215113812
['2021525323333', '등록된강아지', '1.0']
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
20215114214
['2021525323333', '등록된강아지', '1.0']
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
20215114938
['a', '미등록강아지', '0.906422099328031']
hi
20215115941
['2021525323333', '등록된강아지', '0.9640837477118156']
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
2021512110
['2021525323333', '등록된강아지', '0.836461384980017']
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
2021512553
['2021525323333', '등록된강아지', '0.8240518098590938']
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
20215121026
['2021525493333', '등록된강아지', '0.6124057786472163']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121045
['2021525493333', '등록된강아지', '0.6124057786472163']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121145
['2021525493333', '등록된강아지', '0.6124057786472163']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121232
['2021525323333', '등록된강아지', '0.9293305991139946']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121254
['2021525323333', '등록된강아지', '0.9145641751051218']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121415
['2021525493333', '등록된강아지', '0.6124057786472163']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121537
['a', '미등록강아지', '0.4083875823682833']
미등록강아지
2021512164
['2021525323333', '등록된강아지', '0.9201257609426198']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121830
['2021525323333', '등록된강아지', '0.9205804852808248']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121930
['2020420304321', '등록된강아지', '0.705913044533071']
등록된강아지
lookupdb에 예외가 발생했습니다
20215121956
['2020420304321', '등록된강아지', '0.9071640173491189']
등록된강아지
lookupdb에 예외가 발생했습니다
20215122017
['2020420304321', '등록된강아지', '0.9382088534544627']
등록된강아지
lookupdb에 예외가 발생했습니다
20215122034
['2020420304321', '등록된강아지', '0.8063359411047296']
등록된강아지
lookupdb에 예외가 발생했습니다
20215122912
['2021525323333', '등록된강아지', '0.8759861667235264']
등록된강아지
lookupdb에 예외가 발생했습니다
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:42:11] "POST /lookup HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 12:42:12] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:43:10] "POST /lookup HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 12:43:11] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/SVM-Classifier/Classifier.py', reloading
20215124159
['2021525323333', '등록된강아지', '0.9097966658075214']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
20215124258
['2021525323333', '등록된강아지', '0.8925973148043161']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
175.196.169.199 - - [25/May/2021 12:48:53] "POST /lookup HTTP/1.1" 200 -
175.196.169.199 - - [25/May/2021 12:48:54] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [25/May/2021 13:54:52] "POST /register HTTP/1.1" 200 -
220.127.26.124 - - [25/May/2021 13:54:52] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [25/May/2021 13:57:43] "POST /register HTTP/1.1" 200 -
220.127.26.124 - - [25/May/2021 13:57:44] "GET /static/img/2020101201234.jpg HTTP/1.1" 304 -
185.100.87.136 - - [25/May/2021 14:07:13] code 400, message Bad HTTP/0.9 request type ('\x00\x00\x00\x11\x00\x00\x01é\x00\x01\x00\x00\x00ÿÿÿÿ\x01\x00\x00\x00\x00\x00\x00\x00')
185.100.87.136 - - [25/May/2021 14:07:13] "     é    ÿÿÿÿ          MyCustomAssembly   " HTTPStatus.BAD_REQUEST -
162.142.125.128 - - [25/May/2021 14:43:03] "GET / HTTP/1.1" 200 -
209.17.96.138 - - [25/May/2021 18:39:32] "GET / HTTP/1.1" 200 -
162.142.125.54 - - [25/May/2021 22:04:00] code 400, message Bad request version ('À\x14À')
162.142.125.54 - - [25/May/2021 22:04:00] " {  wÒ}U6°üìÌG£É$`÷§¸åóDAö  À/À+ÀÀÀÀ	ÀÀ" HTTPStatus.BAD_REQUEST -
162.142.125.54 - - [25/May/2021 22:04:01] "GET / HTTP/1.1" 200 -
162.142.125.54 - - [25/May/2021 22:04:01] "GET / HTTP/1.1" 200 -
209.17.97.58 - - [25/May/2021 22:13:31] "GET / HTTP/1.0" 200 -
162.142.125.53 - - [26/May/2021 02:54:33] code 400, message Bad request version ('À\x14À')
162.142.125.53 - - [26/May/2021 02:54:33] " {  wYßð ûõXoJù¼ó±Æér ÌH¦4óôÖF¬  À/À+ÀÀÀÀ	ÀÀ" HTTPStatus.BAD_REQUEST -
162.142.125.53 - - [26/May/2021 02:54:34] "GET / HTTP/1.1" 200 -
162.142.125.53 - - [26/May/2021 02:54:34] "GET / HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:29:29] "POST /register HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:29:30] "GET /static/img/2021525323333.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:29:52] "POST /lookup HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 263, in lookup
    lookupimg.save('./SVM-Classifier/Dog-Data/test/%s.jpg' %str(formomentLookup))
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/werkzeug/datastructures.py", line 3013, in save
    dst = open(dst, "wb")
FileNotFoundError: [Errno 2] No such file or directory: './SVM-Classifier/Dog-Data/test/2021532952.jpg'
220.127.26.124 - - [26/May/2021 03:30:00] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:30:01] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:30:26] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:30:27] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:33:32] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:33:32] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:35:32] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:36:23] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:36:23] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:38:52] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:38:53] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:41:07] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:41:07] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:41:23] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:41:23] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:43:36] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:43:36] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 03:45:13] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 03:45:13] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
167.248.133.39 - - [26/May/2021 03:53:13] code 400, message Bad request version ('À\x14À')
167.248.133.39 - - [26/May/2021 03:53:13] " {  wBCÅÈþ?×äz<e%Ö.vrqlÍÍÜö2ñµ  À/À+ÀÀÀÀ	ÀÀ" HTTPStatus.BAD_REQUEST -
167.248.133.39 - - [26/May/2021 03:53:14] "GET / HTTP/1.1" 200 -
167.248.133.39 - - [26/May/2021 03:53:15] "GET / HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 04:29:49] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 04:30:02] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [26/May/2021 04:30:03] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [26/May/2021 04:30:20] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
166.104.245.98 - - [26/May/2021 04:52:33] "POST /lookup HTTP/1.1" 200 -
166.104.245.98 - - [26/May/2021 04:52:34] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
162.142.125.39 - - [26/May/2021 07:58:16] code 400, message Bad request version ('À\x14À')
162.142.125.39 - - [26/May/2021 07:58:16] " {  wÀ¢»ÒUVÞÑ¶Y8îüEî|HS[øâd5  À/À+ÀÀÀÀ	ÀÀ" HTTPStatus.BAD_REQUEST -
162.142.125.39 - - [26/May/2021 07:58:17] "GET / HTTP/1.1" 200 -
162.142.125.39 - - [26/May/2021 07:58:17] "GET / HTTP/1.1" 200 -
104.140.188.2 - - [26/May/2021 09:19:32] "GET / HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:16:39] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 10:16:40] "GET /static/img/2021525323333.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:22:26] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 10:22:27] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:25:02] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 10:25:04] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/SVM-Classifier/Classifier.py', reloading
20215124841
['2021525323333', '등록된강아지', '0.9589613077338446']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
20215135440
['2020101201234', '등록된강아지', '0.7616987519570011']
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
20215135731
['2020101201234', '등록된강아지', '0.748898667270009']
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
2021532917
['2021525323333', '등록된강아지', '0.869047609490638']
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
2021532948
2021532952
['2021525323333', '등록된강아지', '0.8634527692758202']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021533014
['2021525323333', '등록된강아지', '0.8371540660586757']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021533320
['2021525323333', '등록된강아지', '0.8618926752430014']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021533520
['2021525323333', '등록된강아지', '0.8431585058622725']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021533611
['2021525323333', '등록된강아지', '0.8531827456735517']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021533840
['2021525323333', '등록된강아지', '0.864948195096052']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021534055
['2021525323333', '등록된강아지', '0.8094053510599197']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021534111
['2021525323333', '등록된강아지', '0.8540133523721305']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021534324
['2021525323333', '등록된강아지', '0.8539124491307237']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
202153451
['2021525323333', '등록된강아지', '0.8581763820129918']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021542937
['2021525323333', '등록된강아지', '0.8424030421633485']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021542950
['2021525323333', '등록된강아지', '0.8567278260800859']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
202154308
['2021525323333', '등록된강아지', '0.8591230344552198']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021545221
['2020101201234', '등록된강아지', '0.9031827250891715']
등록된강아지
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
((19, '문수림', '01056501234', 'msr@gmail.com'),)
20215101627
['2021525323333', '등록된강아지', '0.9113123013185324']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
20215102214
['2020101201234', '등록된강아지', '0.7895848738503978']
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
20215102450
['2020101201234', '등록된강아지', '1.0']
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:50:08] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 10:50:09] "GET /static/img/2020420304321.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:51:57] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 10:51:58] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:55:50] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:56:52] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 10:56:54] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:58:17] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 10:59:44] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:00:11] "POST /lookup HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 263, in lookup
    lookupimg.save('./SVM-Classifier/Dog-Data/test/%s.jpg' %str(formomentLookup))
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/werkzeug/datastructures.py", line 3013, in save
    dst = open(dst, "wb")
FileNotFoundError: [Errno 2] No such file or directory: './SVM-Classifier/Dog-Data/test/2021511011.jpg'
61.77.60.206 - - [26/May/2021 11:00:19] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:02:56] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:02:57] "GET /static/img/2021526555432.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:03:34] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:04:23] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:04:24] "GET /static/img/2021526555432.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:11:38] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:11:39] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:13:03] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:13:04] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:27:40] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:27:41] "GET /static/img/202152697465.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:28:30] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:29:25] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:29:26] "GET /static/img/2021526968424.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:29:51] "POST /lookup HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 263, in lookup
    lookupimg.save('./SVM-Classifier/Dog-Data/test/%s.jpg' %str(formomentLookup))
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/werkzeug/datastructures.py", line 3013, in save
    dst = open(dst, "wb")
FileNotFoundError: [Errno 2] No such file or directory: './SVM-Classifier/Dog-Data/test/20215112951.jpg'
61.77.60.206 - - [26/May/2021 11:29:55] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:29:56] "GET /static/img/202152697465.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:31:27] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:31:28] "GET /static/img/2021526968424.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:33:58] "POST /lookup HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 11:33:59] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 11:50:52] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 12:17:51] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 12:17:51] "GET /static/img/202152686.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 12:19:41] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 12:19:42] "GET /static/img/20215266.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 12:22:33] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 12:22:33] "GET /static/img/20215266.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.60.206 - - [26/May/2021 12:50:28] "POST /register HTTP/1.1" 200 -
61.77.60.206 - - [26/May/2021 12:50:29] "GET /static/img/2021526541746.jpg HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/YOLOv5/models/experimental.py', reloading
20215104952
['2020420304321', '등록된강아지', '0.7978390252133211']
2020420304321
((32, '차우', '진돗개', '2020', '남', 'profileImg/2020420304321', 19, '2020420304321'),)
20215105140
['2020101201234', '등록된강아지', '0.7988841694314608']
등록된강아지
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
((19, '문수림', '01056501234', 'msr@gmail.com'),)
20215105534
['a', '미등록강아지', '0.30539635077259436']
미등록강아지
20215105636
['2020101201234', '등록된강아지', '0.5944121692983383']
등록된강아지
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
((19, '문수림', '01056501234', 'msr@gmail.com'),)
2021510581
['a', '미등록강아지', '0.6038695793825991']
미등록강아지
20215105928
['a', '미등록강아지', '0.44445080333081577']
미등록강아지
202151103
2021511011
['a', '미등록강아지', '0.39903320293813066']
미등록강아지
2021511239
['a', '미등록강아지', '0.5553528882346657']
hi
2021511314
['a', '미등록강아지', '0.41748497116479216']
미등록강아지
202151143
['2021526555432', '등록된강아지', '0.878819166979693']
등록된강아지
2021526555432
((35, '리트', '진돗개', '2019', '여', 'profileImg/2021526555432', 21, '2021526555432'),)
((21, '이수정', '01098765432', 'lsj@gmail.com'),)
20215111119
['2020101201234', '등록된강아지', '0.7132484257771732']
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
20215111244
['2020101201234', '등록된강아지', '0.6072806168450285']
등록된강아지
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
((19, '문수림', '01056501234', 'msr@gmail.com'),)
20215112721
['a', '미등록강아지', '0.42532733848271764']
hi
2021511287
['a', '미등록강아지', '0.5556515318171851']
미등록강아지
2021511292
['a', '미등록강아지', '0.6044878772097414']
hi
20215112930
20215112951
['202152697465', '등록된강아지', '0.7953318605827306']
등록된강아지
202152697465
((36, '바둑', '치와와', '2018', '여', 'profileImg/202152697465', 22, '202152697465'),)
((22, '김이랑', '01033337465', 'kl@gmail.com'),)
2021511312
['2021526968424', '등록된강아지', '1.0']
등록된강아지
2021526968424
((37, '민주', '포메라니안', '2018', '여', 'profileImg/2021526968424', 23, '2021526968424'),)
((23, '김종훈', '01044778424', 'iproj2@1.2'),)
20215113334
['2020101201234', '등록된강아지', '0.8751426335128287']
등록된강아지
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
((19, '문수림', '01056501234', 'msr@gmail.com'),)
20215115027
['a', '미등록강아지', '0.7533579194451097']
미등록강아지
20215121726
['a', '미등록강아지', '0.6076696119163499']
hi
20215121915
['a', '미등록강아지', '0.5061981874791692']
hi
2021512225
['20215266', '등록된강아지', '1.0']
20215266
((39, '초코', '치와와', '2017', '여', 'profileImg/20215266', 24, '20215266'),)
2021512500
['a', '미등록강아지', '0.6295498284008175']
hi
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [26/May/2021 13:00:38] "POST /register HTTP/1.1" 200 -
106.101.0.82 - - [26/May/2021 13:00:39] "GET /static/img/202152697465.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [26/May/2021 13:43:59] "POST /lookup HTTP/1.1" 200 -
106.101.0.82 - - [26/May/2021 13:44:01] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [26/May/2021 13:56:11] "POST /lookup HTTP/1.1" 200 -
106.101.0.82 - - [26/May/2021 13:56:14] "GET /static/img/2021526541746.jpg HTTP/1.1" 304 -
185.173.35.33 - - [26/May/2021 17:08:35] "GET / HTTP/1.0" 200 -
209.17.96.34 - - [26/May/2021 23:43:54] "GET / HTTP/1.1" 200 -
209.17.96.114 - - [27/May/2021 01:36:14] "GET / HTTP/1.0" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [27/May/2021 02:07:07] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [27/May/2021 02:07:08] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [27/May/2021 02:07:57] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [27/May/2021 02:07:58] "GET /static/img/2021525323333.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [27/May/2021 04:49:59] "POST /register HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 100, in register
    shutil.copyfile(source, destination)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/shutil.py", line 121, in copyfile
    with open(dst, 'wb') as fdst:
FileNotFoundError: [Errno 2] No such file or directory: './SVM-Classifier/rawimage/202152759593461/0.jpg'
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
202151307
['202152697465', '등록된강아지', '0.9028108000917117']
202152697465
((36, '바둑', '치와와', '2018', '여', 'profileImg/202152697465', 22, '202152697465'),)
20215134328
['2020101201234', '등록된강아지', '0.6083450272251455']
등록된강아지
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
((19, '문수림', '01056501234', 'msr@gmail.com'),)
20215135540
['2021526541746', '등록된강아지', '0.6247620176506917']
등록된강아지
2021526541746
((40, '욜로', '포메라니안', '2021', '여', 'profileImg/2021526541746', 25, '2021526541746'),)
((25, '김두호', '01046361746', 'kdh@gmail.com'),)
202152637
['2021525323333', '등록된강아지', '0.6683025543769174']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
202152726
['2021525323333', '등록된강아지', '0.6457852565695551']
등록된강아지
2021525323333
((33, '왕코', '치와와', '2018', '여', 'profileImg/2021525323333', 20, '2021525323333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021544928
['a', '미등록강아지', '0.4989058863476026']
hi
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [27/May/2021 04:55:30] "POST /register HTTP/1.1" 200 -
106.101.0.82 - - [27/May/2021 04:55:33] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [27/May/2021 04:56:39] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
python: can't open file '../YOLOv5/detect.py': [Errno 2] No such file or directory
106.101.0.82 - - [27/May/2021 04:58:36] "POST /register HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 108, in register
    os.system('python ../YOLOv5/detect.py --source rawimage/%s --weights ../YOLOv5/best.pt --conf 0.25') %(reg_num)
TypeError: unsupported operand type(s) for %: 'int' and 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 111, in register
    return jsonify({'messgae:fail'})
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 348, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 129, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 56, in default
    return super().default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type set is not JSON serializable
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021545458
['2020101201234', '등록된강아지', '0.8070159414673252']
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
202154568
['a', '미등록강아지', '0.32687133586030315']
미등록강아지
202154585
['a', '미등록강아지', '0.40311055431580856']
hi
[등록] 미등록 강아지 yolo 코드 예외 발생
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
sh: line 0: cd: ../YOLOv5: No such file or directory
106.101.0.82 - - [27/May/2021 05:14:32] "POST /register HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 108, in register
    os.system('cd ../YOLOv5 && python detect.py --source rawimage/%s --weights ../YOLOv5/best.pt --conf 0.25') %(reg_num)
TypeError: unsupported operand type(s) for %: 'int' and 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 111, in register
    return jsonify({'messgae:fail'})
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 348, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 129, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 56, in default
    return super().default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type set is not JSON serializable
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021551358
['a', '미등록강아지', '0.3951031249594881']
hi
[등록] 미등록 강아지 yolo 코드 예외 발생
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
Traceback (most recent call last):
  File "detect.py", line 6, in <module>
    import torch
ModuleNotFoundError: No module named 'torch'
106.101.0.82 - - [27/May/2021 05:17:04] "POST /register HTTP/1.1" 500 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
Traceback (most recent call last):
  File "detect.py", line 6, in <module>
    import torch
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/__init__.py", line 195, in <module>
    _load_global_deps()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/__init__.py", line 148, in _load_global_deps
    ctypes.CDLL(lib_path, mode=ctypes.RTLD_GLOBAL)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/ctypes/__init__.py", line 364, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/lib/libtorch_global_deps.so: cannot open shared object file: No such file or directory
Traceback (most recent call last):
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 108, in register
    os.system('cd YOLOv5 && python detect.py --source rawimage/%s --weights ../YOLOv5/best.pt --conf 0.25') %(reg_num)
TypeError: unsupported operand type(s) for %: 'int' and 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 111, in register
    return jsonify({'messgae:fail'})
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 348, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 129, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 56, in default
    return super().default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type set is not JSON serializable
106.101.0.82 - - [27/May/2021 05:29:36] "POST /register HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 108, in register
    os.system('cd YOLOv5 && python detect.py --source rawimage/%s --weights ../YOLOv5/best.pt --conf 0.25') %(reg_num)
TypeError: unsupported operand type(s) for %: 'int' and 'str'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 111, in register
    return jsonify({'messgae:fail'})
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 348, in jsonify
    f"{dumps(data, indent=indent, separators=separators)}\n",
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 129, in dumps
    rv = _json.dumps(obj, **kwargs)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 201, in encode
    chunks = list(chunks)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 438, in _iterencode
    o = _default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/json/__init__.py", line 56, in default
    return super().default(o)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type set is not JSON serializable
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021551632
['a', '미등록강아지', '0.5889532948525453']
hi
[등록] 미등록 강아지 yolo 코드 예외 발생
202155293
['a', '미등록강아지', '0.5038781246053144']
hi
[등록] 미등록 강아지 yolo 코드 예외 발생
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
Traceback (most recent call last):
  File "detect.py", line 6, in <module>
    import torch
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/__init__.py", line 195, in <module>
    _load_global_deps()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/__init__.py", line 148, in _load_global_deps
    ctypes.CDLL(lib_path, mode=ctypes.RTLD_GLOBAL)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/ctypes/__init__.py", line 364, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/lib/libtorch_global_deps.so: cannot open shared object file: No such file or directory
106.101.0.82 - - [27/May/2021 05:39:26] "POST /register HTTP/1.1" 200 -
106.101.0.82 - - [27/May/2021 05:39:27] "GET /static/img/202152726687548.jpg HTTP/1.1" 200 -
 * Serving Flask app 'main' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://172.31.34.126:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
61.77.61.43 - - [27/May/2021 11:56:00] "POST /lookup HTTP/1.1" 200 -
61.77.61.43 - - [27/May/2021 11:56:45] "GET /static/img/2021525493333.jpg HTTP/1.1" 200 -
61.77.61.43 - - [27/May/2021 11:56:46] "GET /favicon.ico HTTP/1.1" 404 -
61.77.61.43 - - [27/May/2021 11:59:41] "POST /register HTTP/1.1" 500 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='rawimage/202152731344939', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Traceback (most recent call last):
  File "detect.py", line 188, in <module>
    detect()
  File "detect.py", line 33, in detect
    model = attempt_load(weights, map_location=device)  # load FP32 model
  File "/home/ec2-user/capstone/Save-Pets-Server/YOLOv5/models/experimental.py", line 118, in attempt_load
    model.append(torch.load(w, map_location=map_location)['model'].float().fuse().eval())  # load FP32 model
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/serialization.py", line 579, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: '../YOLOv5/best.pt'
Traceback (most recent call last):
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2069, in __call__
    return self.wsgi_app(environ, start_response)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2054, in wsgi_app
    response = self.handle_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 2051, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1501, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1499, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/flask/app.py", line 1485, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 39, in register
    profile = request.files['dogProfile']
  File "/home/ec2-user/miniconda3/envs/savepets/lib/python3.7/site-packages/werkzeug/datastructures.py", line 377, in __getitem__
    raise exceptions.BadRequestKeyError(key)
werkzeug.exceptions.BadRequestKeyError: 400 Bad Request: The browser (or proxy) sent a request that this server could not understand.
KeyError: 'dogProfile'
61.77.61.43 - - [27/May/2021 12:00:33] "POST /register HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/YOLOv5/models/experimental.py', reloading
20215115540
['2021525493333', '등록된강아지', '0.6979249555405402']
등록된강아지
2021525493333
((34, '오리', '진돗개', '2017', '여', 'profileImg/2021525493333', 20, '2021525493333'),)
((20, '이해석', '01022223333', 'haeok@gmail.com'),)
2021512011
['a', '미등록강아지', '0.30672182960037375']
hi
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='rawimage/202152735744939', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
Traceback (most recent call last):
  File "detect.py", line 188, in <module>
    detect()
  File "detect.py", line 55, in detect
    dataset = LoadImages(source, img_size=imgsz, stride=stride)
  File "/home/ec2-user/capstone/Save-Pets-Server/YOLOv5/utils/datasets.py", line 132, in __init__
    raise Exception(f'ERROR: {p} does not exist')
Exception: ERROR: /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/rawimage/202152735744939 does not exist
61.77.61.43 - - [27/May/2021 12:27:37] "POST /register HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
20215122715
['a', '미등록강아지', '0.30672182960037375']
hi
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/rawimage/202152712644939', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152712644939/0.jpg: 512x640 1 nose, 2 nostrilss, Done. (0.301s)
image 2/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152712644939/1.jpg: 512x640 1 nose, 2 nostrilss, Done. (0.221s)
image 3/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152712644939/2.jpg: 512x640 1 nose, 2 nostrilss, Done. (0.262s)
image 4/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152712644939/3.jpg: 512x640 1 nose, 2 nostrilss, Done. (0.221s)
image 5/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152712644939/4.jpg: 512x640 1 nose, 2 nostrilss, Done. (0.219s)
Results saved to runs/detect/exp2
Done. (1.357s)
61.77.61.43 - - [27/May/2021 12:35:15] "POST /register HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/rawimage/20215273094939', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215273094939/0.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.174s)
image 2/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215273094939/1.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.166s)
image 3/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215273094939/2.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.211s)
image 4/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215273094939/3.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.166s)
image 5/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215273094939/4.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.165s)
Results saved to runs/detect/exp3
Done. (1.005s)
61.77.61.43 - - [27/May/2021 12:41:33] "POST /register HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/YOLOv5/detect.py', reloading
20215123452
['a', '미등록강아지', '0.30672182960037375']
hi
20215124110
['a', '미등록강아지', '0.49800931631554235']
hi
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/rawimage/20215270154939', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215270154939/0.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.175s)
image 2/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215270154939/1.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.171s)
image 3/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215270154939/2.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.164s)
image 4/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215270154939/3.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.213s)
image 5/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215270154939/4.jpg: 384x640 1 nose, 2 nostrilss, Done. (0.162s)
Results saved to runs/detect/exp4
Done. (1.032s)
61.77.61.43 - - [27/May/2021 12:54:04] "POST /register HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/rawimage/202152743927463', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152743927463/0.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.213s)
image 2/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152743927463/1.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.202s)
image 3/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152743927463/2.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.202s)
image 4/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152743927463/3.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.248s)
image 5/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152743927463/4.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.199s)
Results saved to runs/detect/exp5
Done. (1.394s)
106.101.0.82 - - [27/May/2021 12:59:47] "POST /register HTTP/1.1" 200 -
106.101.0.82 - - [27/May/2021 12:59:49] "GET /static/img/202152743927463.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/rawimage/202152750444-56', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152750444-56/0.jpg: 640x640 1 nose, 2 nostrilss, Done. (0.331s)
image 2/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152750444-56/1.jpg: 640x640 1 nose, 2 nostrilss, Done. (0.269s)
image 3/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152750444-56/2.jpg: 640x640 1 nose, 2 nostrilss, Done. (0.268s)
image 4/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152750444-56/3.jpg: 640x640 1 nose, 2 nostrilss, Done. (0.314s)
image 5/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/202152750444-56/4.jpg: 640x640 1 nose, 2 nostrilss, Done. (0.272s)
Results saved to runs/detect/exp6
Done. (1.533s)
220.127.26.124 - - [27/May/2021 14:55:54] "POST /register HTTP/1.1" 200 -
220.127.26.124 - - [27/May/2021 14:55:55] "GET /static/img/202152750444-56.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [27/May/2021 14:57:43] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [27/May/2021 14:57:44] "GET /static/img/202152750444-56.jpg HTTP/1.1" 304 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/rawimage/20215279502530', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215279502530/0.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.212s)
image 2/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215279502530/1.jpg: 640x480 2 noses, 2 nostrilss, Done. (0.253s)
image 3/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215279502530/2.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.199s)
image 4/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215279502530/3.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.208s)
image 5/5 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/rawimage/20215279502530/4.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.206s)
Results saved to runs/detect/exp7
Done. (1.241s)
220.127.26.124 - - [27/May/2021 15:45:13] "POST /register HTTP/1.1" 200 -
220.127.26.124 - - [27/May/2021 15:45:13] "GET /static/img/20215279502530.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [27/May/2021 15:46:11] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [27/May/2021 15:47:28] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [27/May/2021 15:47:29] "GET /static/img/202152743927463.jpg HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [27/May/2021 15:48:17] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [27/May/2021 15:50:29] "POST /lookup HTTP/1.1" 200 -
209.17.97.10 - - [27/May/2021 18:29:41] "GET / HTTP/1.1" 200 -
162.142.125.54 - - [27/May/2021 18:53:52] code 400, message Bad request version ('À\x14À')
162.142.125.54 - - [27/May/2021 18:53:52] " {  wïD§Ôåt8MÑëïÀð¦k~¬ªØ;çCf}é  À/À+ÀÀÀÀ	ÀÀ" HTTPStatus.BAD_REQUEST -
162.142.125.54 - - [27/May/2021 18:53:54] "GET / HTTP/1.1" 200 -
162.142.125.54 - - [27/May/2021 18:53:55] "GET / HTTP/1.1" 200 -
209.17.97.10 - - [27/May/2021 21:11:47] "GET / HTTP/1.0" 200 -
104.140.188.54 - - [27/May/2021 22:11:32] "GET / HTTP/1.1" 200 -
193.86.95.34 - - [27/May/2021 22:52:39] "GET / HTTP/1.0" 200 -
193.86.95.34 - - [27/May/2021 22:52:39] "GET / HTTP/1.0" 200 -
193.86.95.34 - - [27/May/2021 22:52:40] "GET / HTTP/1.1" 200 -
193.86.95.34 - - [27/May/2021 22:52:41] code 400, message Bad request version ('gcc/7.2.0"}}')
193.86.95.34 - - [27/May/2021 22:52:41] "{"id":1,"jsonrpc":"2.0","method":"login","params":{"login":"49CYQrmrqP2LyHQdMY26JGiq3M9cxFkiSU9PyfSEiXAPVR3kSWwj8Jdh2pJRPHV1ZTCQGwivo1a6494wNk7iasLpD2VrGrx","pass":"x","agent":"XMRig/2.4.2 (Windows NT 6.1; Win64; x64) libuv/1.14.1 gcc/7.2.0"}}" HTTPStatus.BAD_REQUEST -
193.86.95.34 - - [27/May/2021 22:52:42] code 400, message Bad request version ('gcc/7.3.0"}}')
193.86.95.34 - - [27/May/2021 22:52:42] "{"id":1,"jsonrpc":"2.0","method":"login","params":{"login":"4BrL51JCc9NGQ71kWhnYoDRffsDZy7m1HUU7MRU4nUMXAHNFBEJhkTZV9HdaL4gfuNBxLPc3BeMkLGaPbF5vWtANQr2cM6dRYBvTiv1U3V","pass":"x","agent":"XMRig/2.5.0 (Windows NT 6.1; Win64; x64) libuv/1.19.2 gcc/7.3.0"}}" HTTPStatus.BAD_REQUEST -
193.86.95.34 - - [27/May/2021 22:52:43] code 400, message Bad request version ('"mining.subscribe"}')
193.86.95.34 - - [27/May/2021 22:52:43] "{"params": ["ethminer-0.16.0.dev2-10+commit.50e91602", "EthereumStratum/1.0.0"], "id": 1, "method": "mining.subscribe"}" HTTPStatus.BAD_REQUEST -
193.93.62.130 - - [28/May/2021 03:07:57] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
193.93.62.130 - - [28/May/2021 03:07:57] "  /*à     Cookie: mstshash=Administr" HTTPStatus.BAD_REQUEST -
193.93.62.130 - - [28/May/2021 04:14:59] code 400, message Bad HTTP/0.9 request type ('\x03\x00\x00/*à\x00\x00\x00\x00\x00Cookie:')
193.93.62.130 - - [28/May/2021 04:14:59] "  /*à     Cookie: mstshash=Administr" HTTPStatus.BAD_REQUEST -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
39.7.55.36 - - [28/May/2021 09:22:32] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
39.7.55.36 - - [28/May/2021 09:23:33] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
39.7.55.36 - - [28/May/2021 09:26:39] "POST /lookup HTTP/1.1" 200 -
39.7.55.36 - - [28/May/2021 09:26:41] "GET /static/img/202152750444-56.jpg HTTP/1.1" 304 -
104.206.128.2 - - [28/May/2021 10:08:45] "GET / HTTP/1.0" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
20215125341
['a', '미등록강아지', '0.5383242687974009']
hi
20215125920
['a', '미등록강아지', '0.3017692930330335']
hi
20215145524
['a', '미등록강아지', '0.4893005333148789']
hi
20215145715
['202152750444-56', '등록된강아지', '0.7087937636223985']
등록된강아지
202152750444-56
((48, '두부', '포메라니안', '2020', '남', 'profileImg/202152750444-56', 29, '202152750444-56'),)
((29, '김종훈', '010-1234-5678', 'doobu@gmail.com'),)
20215154441
['a', '미등록강아지', '0.4598302380923709']
hi
20215154540
['a', '미등록강아지', '0.3033185624114151']
미등록강아지
20215154657
['202152743927463', '등록된강아지', '0.8034713093069562']
등록된강아지
202152743927463
((47, '킁킁', '치와와', '2020', '여', 'profileImg/202152743927463', 28, '202152743927463'),)
((28, '이지은', '01047337463', 'ljy@gmail.com'),)
20215154746
['a', '미등록강아지', '0.4008574903342162']
미등록강아지
20215154958
['a', '미등록강아지', '0.40518492803403244']
미등록강아지
202159221
['a', '미등록강아지', '0.29607375150882553']
미등록강아지
202159232
['a', '미등록강아지', '0.3912062188481992']
미등록강아지
202159269
['202152750444-56', '등록된강아지', '0.9011162842135796']
등록된강아지
202152750444-56
((48, '두부', '포메라니안', '2020', '남', 'profileImg/202152750444-56', 29, '202152750444-56'),)
((29, '김종훈', '010-1234-5678', 'doobu@gmail.com'),)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 10:55:37] "POST /lookup HTTP/1.1" 200 -
106.101.0.82 - - [28/May/2021 10:56:07] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/YOLOv5/utils/wandb_logging/wandb_utils.py', reloading
20215105537
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
2021510567
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 11:00:48] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021511048
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 11:03:24] "POST /lookup HTTP/1.1" 200 -
106.101.0.82 - - [28/May/2021 11:04:28] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021511324
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
2021511428
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 11:07:34] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021511734
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 11:14:55] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
20215111455
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 11:15:36] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
20215111536
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 [Errno 2] No such file or directory: './SVM-Classifier/testimage/%s/%s.jpg'
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/2021511242', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/2021511242/0.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.211s)
Results saved to runs/detect/exp
Done. (0.282s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
Traceback (most recent call last):
  File "Classifier.py", line 167, in <module>
    main()
  File "Classifier.py", line 116, in main
    img_test = histo_clahe(opt.dir + '/test/' + opt.test)
  File "/home/ec2-user/capstone/Save-Pets-Server/SVM-Classifier/histo_clahe.py", line 7, in histo_clahe
    height, width, channel = img.shape
AttributeError: 'NoneType' object has no attribute 'shape'
106.101.0.82 - - [28/May/2021 11:24:35] "POST /lookup HTTP/1.1" 200 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215112621', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215112621/0.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.215s)
Results saved to runs/detect/exp2
Done. (0.288s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
Traceback (most recent call last):
  File "Classifier.py", line 167, in <module>
    main()
  File "Classifier.py", line 116, in main
    img_test = histo_clahe(opt.dir + '/test/' + opt.test)
  File "/home/ec2-user/capstone/Save-Pets-Server/SVM-Classifier/histo_clahe.py", line 7, in histo_clahe
    height, width, channel = img.shape
AttributeError: 'NoneType' object has no attribute 'shape'
106.101.0.82 - - [28/May/2021 11:26:54] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/YOLOv5/detect.py', reloading
2021511242
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215112621
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215113415', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215113415/0.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.261s)
Results saved to ../SVM-Classifier/Dog-Data/test/0.jpg
Done. (0.331s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
Traceback (most recent call last):
  File "Classifier.py", line 167, in <module>
    main()
  File "Classifier.py", line 116, in main
    img_test = histo_clahe(opt.dir + '/test/' + opt.test)
  File "/home/ec2-user/capstone/Save-Pets-Server/SVM-Classifier/histo_clahe.py", line 7, in histo_clahe
    height, width, channel = img.shape
AttributeError: 'NoneType' object has no attribute 'shape'
106.101.0.82 - - [28/May/2021 11:34:48] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
20215113415
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 11:36:00] "POST /lookup HTTP/1.1" 200 -
106.101.0.82 - - [28/May/2021 11:37:09] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
2021511360
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
2021511379
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 not enough arguments for format string
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
106.101.0.82 - - [28/May/2021 11:38:19] "POST /lookup HTTP/1.1" 200 -
106.101.0.82 - - [28/May/2021 11:38:27] "POST /lookup HTTP/1.1" 200 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
20215113819
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 decoding str is not supported
20215113827
[조회]yolov5가 작동하지 않아 조회가 되지 않습니다 decoding str is not supported
 * Restarting with stat
  File "/home/ec2-user/capstone/Save-Pets-Server/main.py", line 285
    os.system('cd YOLOv5 && python detect.py --source ../SVM-Classifier/testimage/%s --weights ../YOLOv5/best.pt --option test --conf 0.25' %str(formomentLookup))
     ^
SyntaxError: invalid syntax
 * Serving Flask app 'main' (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://172.31.34.126:5000/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215122159', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215122159/20215122159.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.219s)
Results saved to ../SVM-Classifier/Dog-Data/test/20215122159.jpg
Done. (0.287s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [28/May/2021 12:22:33] "POST /lookup HTTP/1.1" 200 -
106.101.0.82 - - [28/May/2021 12:22:34] "GET /static/img/2020101201234.jpg HTTP/1.1" 200 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/2021512235', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/2021512235/2021512235.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.212s)
Results saved to ../SVM-Classifier/Dog-Data/test/2021512235.jpg
Done. (0.247s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [28/May/2021 12:23:39] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [28/May/2021 12:23:40] "GET /static/img/20215279502530.jpg HTTP/1.1" 304 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/2021512249', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/2021512249/2021512249.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.213s)
Results saved to ../SVM-Classifier/Dog-Data/test/2021512249.jpg
Done. (0.248s)
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:12] "POST /lookup HTTP/1.1" 200 -
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:14] "POST /lookup HTTP/1.1" 200 -
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:18] "POST /lookup HTTP/1.1" 200 -
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:24] "POST /lookup HTTP/1.1" 200 -
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:33] "POST /lookup HTTP/1.1" 200 -
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:34] "POST /lookup HTTP/1.1" 200 -
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:36] "POST /lookup HTTP/1.1" 200 -
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:37] "POST /lookup HTTP/1.1" 200 -
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:24:39] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [28/May/2021 12:24:43] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [28/May/2021 12:24:44] "GET /static/img/20215279502530.jpg HTTP/1.1" 304 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/2021512250', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/2021512250/2021512250.jpg: 640x640 1 nose, 3 nostrilss, Done. (0.286s)
Results saved to ../SVM-Classifier/Dog-Data/test/2021512250.jpg
Done. (0.306s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [28/May/2021 12:25:33] "POST /lookup HTTP/1.1" 200 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215122634', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215122634/20215122634.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.211s)
Results saved to ../SVM-Classifier/Dog-Data/test/20215122634.jpg
Done. (0.243s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [28/May/2021 12:27:07] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [28/May/2021 12:27:07] "GET /static/img/20215279502530.jpg HTTP/1.1" 304 -
 * Detected change in '/home/ec2-user/capstone/Save-Pets-Server/main.py', reloading
20215122159
['2020101201234', '등록된강아지', '0.5981607960960778']
등록된강아지
2020101201234
((31, '몽', '푸들', '2010', '남', 'profileImg/2020101201234', 19, '2020101201234'),)
((19, '문수림', '01056501234', 'msr@gmail.com'),)
2021512235
['20215279502530', '등록된강아지', '1.0']
등록된강아지
20215279502530
((49, '개코', '진돗개', '2018', '여', 'profileImg/20215279502530', 30, '20215279502530'),)
((30, '이해석', '01072472530', 'haeseok@gmail.com'),)
2021512249
20215122412
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122414
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122418
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122424
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122433
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122434
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122436
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122437
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
20215122439
[조회] ML코드가 작동하지 않아 조회가 되지 않습니다 attribute error
['20215279502530', '등록된강아지', '1.0']
등록된강아지
20215279502530
((49, '개코', '진돗개', '2018', '여', 'profileImg/20215279502530', 30, '20215279502530'),)
((30, '이해석', '01072472530', 'haeseok@gmail.com'),)
2021512250
['a', '미등록강아지', '0.4850899812941249']
미등록강아지
20215122634
['20215279502530', '등록된강아지', '1.0']
등록된강아지
20215279502530
((49, '개코', '진돗개', '2018', '여', 'profileImg/20215279502530', 30, '20215279502530'),)
((30, '이해석', '01072472530', 'haeseok@gmail.com'),)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 498-642-075
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215122841', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215122841/20215122841.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.267s)
Results saved to ../SVM-Classifier/Dog-Data/test/20215122841.jpg
Done. (0.300s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
sh: line 0: cd: YOLOv5: No such file or directory
python: can't open file 'Classifier.py': [Errno 2] No such file or directory
220.127.26.124 - - [28/May/2021 12:28:50] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [28/May/2021 12:29:15] "POST /lookup HTTP/1.1" 200 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215122930', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215122930/20215122930.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.235s)
Results saved to ../SVM-Classifier/Dog-Data/test/20215122930.jpg
Done. (0.290s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [28/May/2021 12:30:04] "POST /lookup HTTP/1.1" 200 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/2021512309', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/2021512309/2021512309.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.215s)
Results saved to ../SVM-Classifier/Dog-Data/test/2021512309.jpg
Done. (0.298s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
106.101.0.82 - - [28/May/2021 12:30:43] "POST /register HTTP/1.1" 200 -
106.101.0.82 - - [28/May/2021 12:30:46] "GET /static/img/20215270154939.jpg HTTP/1.1" 200 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215123253', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215123253/20215123253.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.211s)
Results saved to ../SVM-Classifier/Dog-Data/test/20215123253.jpg
Done. (0.246s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [28/May/2021 12:33:27] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [28/May/2021 12:33:28] "GET /static/img/20215279502530.jpg HTTP/1.1" 304 -
YOLOv5  torch 1.8.1 CPU

Model Summary: 224 layers, 7056607 parameters, 0 gradients
Namespace(agnostic_nms=False, augment=False, classes=None, conf_thres=0.25, device='', exist_ok=False, img_size=640, iou_thres=0.45, name='exp', option='test', project='runs/detect', save_conf=False, save_txt=False, source='../SVM-Classifier/testimage/20215123341', update=False, view_img=False, weights=['../YOLOv5/best.pt'])
Fusing layers... 
image 1/1 /home/ec2-user/capstone/Save-Pets-Server/YOLOv5/../SVM-Classifier/testimage/20215123341/20215123341.jpg: 640x480 1 nose, 2 nostrilss, Done. (0.210s)
Results saved to ../SVM-Classifier/Dog-Data/test/20215123341.jpg
Done. (0.248s)
[ WARN:0] global /tmp/pip-req-build-yw7uvgqm/opencv_contrib/modules/xfeatures2d/misc/python/shadow_sift.hpp (13) SIFT_create DEPRECATED: cv.xfeatures2d.SIFT_create() is deprecated due SIFT tranfer to the main repository. https://github.com/opencv/opencv/issues/16736
220.127.26.124 - - [28/May/2021 12:34:15] "POST /lookup HTTP/1.1" 200 -
220.127.26.124 - - [28/May/2021 12:34:15] "GET /static/img/20215279502530.jpg HTTP/1.1" 304 -
